configs/gotc/cromwell.conf.ctmpl

```diff
@@ -0,0 +1,410 @@
+    // Time in minutes before an entry expires from the docker hashes cache and needs to be fetched again
+    cache-entry-ttl = "20 minutes"
+    // Maximum number of elements to be kept in the cache. If the limit is reached, old elements will be removed from the cache
+    cache-size = 200
+    // How should docker hashes be looked up. Possible values are "local" and "remote"
+    // "local": Lookup hashes on the local docker daemon using the cli
+    // "remote": Lookup hashes on docker hub and gcr
+    method = "remote"
+  }
+}
+
+engine {
+  # This instructs the engine which filesystems are at its disposal to perform any IO operation that it might need.
+  # For instance, WDL variables declared at the Workflow level will be evaluated using the filesystems declared here.
+  # If you intend to be able to run workflows with this kind of declarations:
+  # workflow {
+  #    String str = read_string("gs://bucket/my-file.txt")
+  # }
+  # You will need to provide the engine with a gcs filesystem
+  # Note that the default filesystem (local) is always available.
+  filesystems {
+    gcs {
+      auth = "service-account"
+    },
+    drs {
+      auth = "service-account"
+    },
+    local {
+      enabled: true
+    },
+    http {
+      enabled: true
+    },
+  }
+}
+
+languages {
+  WDL {
+    versions {
+      "draft-2" {
+        language-factory = "languages.wdl.draft2.WdlDraft2LanguageFactory"
+      }
+      "draft-3" {
+        language-factory = "languages.wdl.draft3.WdlDraft3LanguageFactory"
+      }
+    }
+  }
+ }
+
+backend {
+  default = "PAPIv2"
+  providers {
+    PAPIv2 {
+      actor-factory = "cromwell.backend.google.pipelines.v2alpha1.PipelinesApiLifecycleActorFactory"
+      config {
+        slow-job-warning-time: 24 hours
+
+        // Google project
+        project = {{ if eq $env "prod" }}"broad-gotc-{{$env}}"{{else}}"broad-exomes-dev1"{{end}}
+
+        // Base bucket for workflow executions
+        root = "gs://broad-gotc-{{$env}}-cromwell-execution"
+
+        # Set this to the lower of the two values "Queries per 100 seconds" and "Queries per 100 seconds per user" for
+        # your project.
+        #
+        # Used to help determine maximum throughput to the Google Genomics API. Setting this value too low will
+        # cause a drop in performance. Setting this value too high will cause QPS based locks from Google.
+        # 1000 is the default "Queries per 100 seconds per user", 50000 is the default "Queries per 100 seconds"
+        # See https://cloud.google.com/genomics/quotas for more information
+        genomics-api-queries-per-100-seconds = 50000
+
+        # Polling for completion backs-off gradually for slower-running jobs.
+        # This is the maximum polling interval (in seconds):
+        maximum-polling-interval = 600
+
+        # This limit is based on empirical measures of Cromwell's throughput. Identical to FireCloud as of 2019-09-04
+        concurrent-job-limit = 14400
```

Comment:
FC splits jobs across multiple projects (and controls per-project jobs with hog-factors) - for GOTC prod I think we'd probably want this to be `2400` with a hog factor of `1`?